packages/components/package.json

```diff
@@ -24,10 +24,12 @@
     "url": "https://github.com/cultureamp/kaizen-discourse/issues"
   },
   "homepage": "https://github.com/cultureamp/kaizen-design-system#readme",
-  "peerDependencies": {
-    "react": "^16.14.0 || ^17.0.0 || ^18.0.0"
-  },
   "dependencies": {
+    "@kaizen/component-base": "^1.1.2",
```

Comment:
There's honestly almost nothing in this (I think there's only the `OverrideClassName` type), so if you want to just copy/paste the contents in I'm okay with that 😬